notebooks/en/enterprise_cookbook_dev_spaces.md

```diff
@@ -0,0 +1,88 @@
+
+
+
+
+
+## 2. Dev Mode, or: Develop on 4 GPUs from your local VS Code
+
+What if you don't like working in JupyterLab in the browser? Enter `Dev Mode`. `Dev Mode` enables you to SSH into any Space's hardware from their local IDE like VS Code. [HF Pro/Enterprise](https://huggingface.co/pricing) subscribers can activate `Dev Mode` for any Space in the Space's settings. 
+
+Once `Dev Mode` is activated, you will see a pop-up at the bottom left of your JupyterLab Space's window. To SSH into your local VS Code, you first need to install the [VS Code Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) locally and add your SSH key to your [HF Profile](https://huggingface.co/settings/keys). Clicking on `Connect with VS Code` should then open your local VS Code window and establish the remote connection to your space. A similar process should be possible with any IDE that supports remote development with SSH. 
+
+<img src="https://cdn-lfs-us-1.huggingface.co/repos/13/3d/133d8ca2460bf82ba2bdbe928d91a6c780364a6d0cf9005087db081cca492c02/f462b2fa4e25380194c0cf062f1e68d50290d6b1737b357449622491fe756474?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27enterprise-jupyterlab-devmode-popup.png%3B+filename%3D%22enterprise-jupyterlab-devmode-popup.png%22%3B&response-content-type=image%2Fpng&Expires=1717923752&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNzkyMzc1Mn19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy11cy0xLmh1Z2dpbmdmYWNlLmNvL3JlcG9zLzEzLzNkLzEzM2Q4Y2EyNDYwYmY4MmJhMmJkYmU5MjhkOTFhNmM3ODAzNjRhNmQwY2Y5MDA1MDg3ZGIwODFjY2E0OTJjMDIvZjQ2MmIyZmE0ZTI1MzgwMTk0YzBjZjA2MmYxZTY4ZDUwMjkwZDZiMTczN2IzNTc0NDk2MjI0OTFmZTc1NjQ3ND9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoifV19&Signature=eAC0IDJcfic%7EUilq-aHshD5ygR7NYWsTa%7ELBhSu%7EPxfL2BL-0dTL1mgJ5QdQfSEldkszgPb0KaKBJaE-071G9aWHRWJXKLN0eodo%7EsfTchKLIZwBgT-KWwRyRouEqCrc-qquGkTWR3VWc5YwKwsoe2Eg1J2Ft-P-brR%7EguRouQo%7EONPqhvLrGwtHkv-hD-qJ8TIBHq%7EkrUuw65AH8oV9dYLJ8rug4gbNu0z9oAv7XbLku5RUlWLi1EhHyrNTq9-RYpXURu%7E0wZx-ip9E7S8yQcYHIh5%7EJdJ-l32KszUAzkmfAEcUIjk4yDQtzna2MW7Db4BbeDbttYBYQ2kT-swQmQ__&Key-Pair-Id=KCD77M1F0VK2B" width="450">
+
+When connecting to your Space with SSH, your default directory will be an empty `/app` directory. You then need to change to the `/data` directory, where all your persistent files (code, data, models etc.) are stored. The `/data` directory is the only directory with guaranteed file persistance across sessions. You can find the files of your Docker container in the `HOME/user/app` directory, in case you want to modify the underlying Docker container. 
+
+> [!TIP]
+> Persisted files in the `/data` directory are currently not automatically backed up. We therefore recommend making backups of your most important files on a regular basis to avoid accidental data loss. 
+
+> [!TIP]
```

Comment:
Maybe explicitly mention this in `### Creating your JupyterLab Space` where Private Spaces is described to avoid redundancy and a back-to-back Tip.